README.md

```
@@ -1,2 +1,112 @@
+| Architecture | Clean Architecture |
+| Module | Single Module |
+| Pattern | MVI |
+| DI | Hilt |
+| Async | Coroutines & Flow |
+| Network | Retrofit2 + OkHttp |
+| Serialization | Kotlinx Serialization |
+| Image | Coil |
+| Local DB | DataStore (Preferences) |
+| Navigation | Compose Navigation |
+
+**클린 아키텍쳐 선정 이유**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)